mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
+function introduceMe(name, age) {
+    return `Hello, my name is ${name} and I am ${age} years old`;
+}
+
 
 function getTotal(a, b) {
-  total = a ++ b;
+  total = a + b;
 
-  return "The total is total";
+  return `The total is ${total}`;
 }
```

Comment:
Optional solution:
function getTotal(a, b) {
 let total = a + b;

  return `The total is 28`;
}

Reply:
Thank you for sharing, Danny. From my understanding we cannot use 28. (28 is used at the bottom for testing).